packages/svelte/src/compiler/migrate/index.js

```diff
@@ -347,7 +350,7 @@ const instance_script = {
 							)
 						: '',
 					optional: !!declarator.init,
-					bindable: binding.updated,
+					bindable: binding.mutated || binding.reassigned,
```

Comment:
ditto. if there's a case where `binding.updated !== (binding.mutated || binding.reassigned)` then that needs to be fixed